### PR TITLE
Symbols in dbx with text input

### PR DESCRIPTION
**Pull request by RiByryn**
Fixed a bug with using special characters in the input field

### CODE REVIEW

File: examples/project/dialog_box/test_2_input_field.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def test_text_input():
 
 @pytest.mark.case_name("Numeric input")
 def test_num_input():
+
```
Comment (xorialexandrov) [inline examples/project/dialog_box/test_2_input_field.py:28]: ?

File: hardpy/hardpy_panel/frontend/package.json [hunk 2]
```diff
@@ -90,4 +90,4 @@
     "eslint": "^8.52.0",
     "eslint-plugin-react-hooks": "^4.6.0"
   }
-}
+}
```
Comment (xorialexandrov) [inline hardpy/hardpy_panel/frontend/package.json:93]: ?

File: examples/project/dialog_box/test_2_input_field.py (hunk 1 above)
Comment (RiByryn) [reply examples/project/dialog_box/test_2_input_field.py]: deleted

File: hardpy/hardpy_panel/frontend/package.json (hunk 2 above)
Comment (RiByryn) [reply hardpy/hardpy_panel/frontend/package.json]: corrected

**Pull request merged**